clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -95,10 +95,9 @@ public class CommitRequestManager implements RequestManager, MemberStateListener
     private Optional<Integer> lastEpochSentOnCommit;
 
     /**
-     *  Latest member ID and epoch received via the {@link #onMemberEpochUpdated(Optional, Optional)},
-     *  to be included in the OffsetFetch and OffsetCommit requests if present. This will have
-     *  the latest values received from the broker, or empty of the member is not part of the
-     *  group anymore.
+     *  Latest member ID and epoch received via the {@link MemberStateListener#onMemberEpochUpdated(Optional, String)},
```

Comment:
nit: the concept of "latest member ID received" is confusing now given that we don't receive it from the broker anymore. Maybe something like "The member ID and latest epoch..." (used in other places here too, let's fix them all)